src/navsat_transform.cpp

```diff
@@ -129,15 +129,17 @@ NavSatTransform::NavSatTransform(const rclcpp::NodeOptions & options) :
     datumCallback(request, response);
   }
 
+  auto custom_qos = rclcpp::QoS(rclcpp::KeepLast(1)).durability_volatile().best_effort();
```

Comment:
Please use the sensor data QoS, don't specify all of this. The sensor data QoS is just given to you for free

Reply:
yeah no excuses, fixed now :)